zerver/lib/subscription_info.py

```diff
@@ -398,7 +426,11 @@ def validate_user_access_to_subscribers_helper(
         raise JsonableError(_("Subscriber data is not available for this channel"))
 
     # Organization administrators can view subscribers for all streams.
```

Comment:
This comment is stale. But I think actually the comment is fine, and instead we should just have an additional block rather than making this a compound `or` statement so we can a nice comment for the new clause.